src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,67 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/no-shadow */
```

Comment:
Using `eslint-disable` is a bag practice. Try to avoid it here and in test tasks. Find the way to solve it without `eslint-disable`.